bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -192,16 +150,7 @@ spec:
               command:
                 - bash
                 - -ec
-                - |
```

Comment:
the readiness probe simply needs to check if postgresql queries can be executed.